app/main.py

```diff
@@ -2,0 +1,35 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name: "str", health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
```

Comment:
Great job on using type annotations for your method parameters and return types. This is a good practice as it makes your code more readable and self-documenting. Keep it up!